spec.md

```diff
@@ -318,6 +318,14 @@ that contains both context and data).
 * Constraints:
   * REQUIRED
   * MUST be a non-empty string
+  
+### ordinal
+* Type: `String`
+* Description: Value expressing the relative order of the event. This enables
```

Comment:
Would we need to say what its relative to?  Also, are there any uniqueness constraints? Could a source just put "1" for all events and still be compliant?

Reply:
@duglin - good call, I should've included that it must be increasing contiguously

Reply:
If you're filtering events, then you need to be projecting to a new stream, something like:
`event -> consumer -> command (incl validation) -> new_event`
because your validation and reprojection changes. 
Contiguous events are simple to do if you're validating events - events can't validate without a validating state, and a validating state represents a projection at an ordinal (..ie so you already need to have the ordinal).

Reply:
Another way of thinking about it is that one can filter commands, but not events.